docs/k6-executors.md

```diff
@@ -0,0 +1,3 @@
+# K6 Executors
+
+For more information on using K6 executors, please visit - <https://k6.io/docs/using-k6/scenarios/executors/>.
```

Comment:
These K6 executors are a different thing than our Testkube executors. I feel it will confuse the users. Lets remove this file.